### PR TITLE
Fixed habitat installation tutorial to use new docker image at proper address

### PR DESCRIPTION
**Pull request by klemenkotar**
- Moved the docker image to `allenact` group on docker hub and update tutorial accordingly
- Added more detailed instructions for downloading the gibson dataset
-  Added the changes luca requested to the docker container and re built it

### CODE REVIEW

File: docs/installation/installation-framework.md [hunk 1]
```diff
@@ -49,25 +49,21 @@ see the above section on installing `iTHOR`.
...
+docker pull allenact/allenact:latest
 ```
 
 This container includes the 0.1.0 release of `allenact`, the 0.1.5 release of `habitat` as well
 as the `Gibson` point navigation dataset. This dataset consists of a set of start and goal positions provided by habitat.
 You then need to launch the container and attach into it:
 
 ```bash
-docker run --runtime=nvidia -it klemenkotar/allenact
+docker run --runtime=nvidia -it allenact/allenact
```
Comment (Lucaweihs) [inline docs/installation/installation-framework.md:60]: While we're here, what happens if I don't have a GPU? Do I just remove `--runtime=nvidia` and everything works? If not, we should definitely have some way of enabling cpu-only training.

File: docs/installation/installation-framework.md [hunk 2]
```diff
@@ -49,25 +49,21 @@ see the above section on installing `iTHOR`.
...
 ```
 
-Once inside the container activate the conda environment:
-
-```bash
-conda activate allenact
-```
+Once inside the container simply cd into the `allenact` directory where all the allenact and habitat code should be stored:
```
Comment (Lucaweihs) [inline docs/installation/installation-framework.md:63]: "cd" -> "`cd`"

**Review by Lucaweihs — COMMENTED** [review]
One minor comment.

**Comment by dhruvbatra** [comment]
FYI -- We are working towards releasing Habitat v0.1.6 next week. As part of it, we will be emphasizing conda nightly builds (see e.g. https://github.com/facebookresearch/habitat-sim/pull/671). 

Installing from conda might be much easier and lightweight than docker.

File: docs/installation/installation-framework.md (hunk 1 above)
Comment (klemenkotar) [reply docs/installation/installation-framework.md:60]: Yes, that is correct we just remove `--runtime=nvidia` but I think that training habitat without a GPU is pretty much pointless. Even evaluations take a very long time. Nonetheless it is worth mentioning.

**Comment by Lucaweihs** [comment]
That's fantastic @dhruvbatra, looking forward to it!

**Comment by klemenkotar** [comment]
That is great to hear @dhruvbatra! We will update the installation guide once your release is live.

**Pull request merged**